assets/src/blocks/Accordion/AccordionFrontend.js

```diff
@@ -1,49 +1,49 @@
 window.dataLayer = window.dataLayer || [];
 
-export const AccordionFrontend = ({ title, description, tabs, className }) => (
-  <section className={`block accordion-block ${className ?? ''}`}>
-    {title &&
-      <header>
-        <h2 className="page-section-header">{title}</h2>
-      </header>
+export const AccordionFrontend = ( { title, description, tabs, className } ) => (
+  <section className={ `block accordion-block ${ className ?? '' }` }>
+    { title &&
+		<header>
+		  <h2 className="page-section-header">{ title }</h2>
+		</header>
     }
-    {description &&
-      <p className="page-section-description" dangerouslySetInnerHTML={{ __html: description }} />
+    { description &&
+		<p className="page-section-description" dangerouslySetInnerHTML={ { __html: description } } />
     }
-    {tabs.map(({ headline, text, button }, index) => {
+    { tabs.map( ( { headline, text, button }, index ) => {
       const { button_text, button_url, button_new_tab } = button || {};
       const buttonProps = {};
-      if (button_new_tab) {
+      if ( button_new_tab ) {
         buttonProps.target = '_blank';
         buttonProps.rel = 'noopener noreferrer';
       }
 
       return (
-        <div key={`accordion-content-${index}`} className='accordion-content'>
-          {headline &&
-            <div
-              className='accordion-headline'
-              name={headline}
-            >
-              {headline}
-            </div>
+        <div key={ `accordion-content-${ index }` } className="accordion-content">
+          { headline &&
+					<div
+					  className="accordion-headline"
+					  name={ headline }
+					>
+					  { headline }
+					</div>
           }
-          <div className='panel panel-hidden'>
-            {text &&
-              <p className="accordion-text" dangerouslySetInnerHTML={{ __html: text }} />
+          <div className="panel panel-hidden">
+            { text &&
+						<p className="accordion-text" dangerouslySetInnerHTML={ { __html: text } } />
             }
-            {button_text &&
-              <a
-                className="btn btn-secondary accordion-btn"
-                href={button_url}
-                {...buttonProps}
-              >
-                {button_text}
-              </a>
+            { button_text &&
+						<a
```

Comment:
I'm curious if this large space normal. Is it? 
I looked the same behavior in other parts of the code.

Reply:
It is not, we will fix this better when we are properly implementing the coding standards.